src/lib/rest/HttpHeaders.h

```diff
@@ -26,6 +26,37 @@
 * Author: Ken Zangelin
 */
 #include <string>
+#include <vector>
+
+#include "common/MimeType.h"
+
+
+
+// -----------------------------------------------------------------------------
```

Comment:
We typically use `/* ... */` for preambles. Not really important, but it has grabed my attention...